source/codegen/common_helpers.py

```diff
@@ -316,6 +327,28 @@ def indent_text_to_level(text, level):
 
 
 def filter_parameters_for_grpc_fields(parameters):
-    """Filter out the parameters that shouldn't be represented by a field on a grpc message.
-       For example, get rid of any parameters whose values should be determined from another parameter."""
-    return [p for p in parameters if p.get('include_in_proto', True)]
+  """Filter out the parameters that shouldn't be represented by a field on a grpc message.
+      For example, get rid of any parameters whose values should be determined from another parameter."""
+  return [p for p in parameters if p.get('include_in_proto', True)]
+
+
+def get_attribute_groups(data):
```

Comment:
Note: this method is key to this change. 

Previously there was always one group and it's name was `${service_class_prefix}`. Now that is the fallback/default.